web/src/components/data-entities/diver/__test__/DriverList.test.tsx

```diff
@@ -0,0 +1,63 @@
+// @ts-ignore
```

Comment:
Should be `DiverList.test.tsx`. Other tests are kept in the same directory as the component instead of in `__tests__`.

Reply:
Fixed typo

Reply:
Removed test